releases/cockroachcloud-04062020.md

```diff
@@ -0,0 +1,26 @@
+---
+title: What&#39;s New in CockroachCloud
+toc: true
+summary: Additions and changes in CockroachCloud.
+build_for: [cockroachcloud]
+---
+
+## April 06, 2020
+
+In addition to various updates, enhancements, and bug fixes, this beta release includes the following major highlights:
+
+- Free trials of CockroachCloud are now available. [Contact us](mailto:sales@cockroachlabs.com) to request a trial code.
+- CockroachCloud now supports VPC peering for clusters running on GCP. [Contact us](https://support.cockroachlabs.com/hc/en-us) to set up a VPC peering-enabled CockroachCloud cluster.
+
+### Security updates
+
+CockroachCloud now requires a user to create a CockroachCloud account before accepting an invite to join an Organization.
```

Comment:
nit: let's say 

> CockroachCloud now requires a user to have a CockroachCloud account...

[Depending on if this is a first-time or existing user, they will either create a new account or verify their existing account. So, I think "have an account" is language that can solve for both cases]